### ISSUE TITLE
Conflict with Jetpack's Testimonials post type

### ISSUE THREAD

**Issue by simontomkins** [WordPress/gutenberg#7634]
## Issue Overview
Cannot edit a page in Gutenberg which contains a testimonial shortcode within it. WordPress instead edits a single testimonial post.

## Steps to Reproduce (for bugs)
1. Install Jetpack.
2. Enable the Testimonials post type.
3. Create a single testimonial post and save.
4. Create a new page using Gutenberg.
5. Add the testimonials shortcode to the page using the shortcodes block, e.g. [testimonials columns=1 display_content=full showposts=1 orderby=date]
6. Save the page and go back to the dashboard.
7. Try and edit that page in Gutenberg.

## Expected Behavior
You should be able to edit the previously created page in Gutenberg.

**Comment by AtrumGeost** [comment]
Hi there,

I tested this with Gutenberg 4.0 and wasn't able to reproduce the issue. It worked properly on my end.

I added the shortcode directly on the text block and also using a shortcode block. Published the page, went to the Dashboard and opened the page again. 

After that, I was able to edit both the text and the shortcode blocks:

![guten gh 7634](https://user-images.githubusercontent.com/3696121/47223103-02bbc400-d37e-11e8-9dd1-142042fb1a6e.gif)

_WordPress 4.9.8 and Gutenberg 4.0.0 using Google Chrome Version 70.0.3538.67 (Official Build) (64-bit) on macOS 10.14 - Mac OS Mojave_
